src/balloontoolbar.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module editor-balloon-toolbar/contextual
+ */
+
+import StandardEditor from '@ckeditor/ckeditor5-core/src/editor/standardeditor';
+import HtmlDataProcessor from '@ckeditor/ckeditor5-engine/src/dataprocessor/htmldataprocessor';
+import ContextualToolbar from '@ckeditor/ckeditor5-ui/src/toolbar/contextual/contextualtoolbar';
+import BalloonToolbarEditorUI from './balloontoolbareditorui';
+import BalloonToolbarEditorUIView from './balloontoolbareditoruiview';
+
+import '../theme/theme.scss';
+
+/**
+ * The balloon toolbar editor. Uses an inline editable and a toolbar based
+ * on the {@link ui/toolbar/contextual/contextualtoolbar~ContextualToolbar}.
+ *
+ * @extends module:core/editor/standardeditor~StandardEditor
+ */
+export default class BalloonToolbarEditor extends StandardEditor {
+	/**
+	 * Creates an instance of the balloon toolbar editor.
+	 *
+	 * @param {HTMLElement} element The DOM element that will be the source for the created editor.
+	 * @param {Object} config The editor configuration.
+	 */
+	constructor( element, config ) {
+		super( element, config );
+
+		this.config.get( 'plugins' ).push( ContextualToolbar );
+		this.config.define( 'contextualToolbar', this.config.get( 'toolbar' ) );
```

Comment:
I am wondering if this will not cause any issues, because it will (correct me if I am wrong) copy reference to same object, so further changes to `toolbar` configuration will also affect `contextualToolbar`. Or maybe this is desired behavior?